HaulageApplication/Haulage/App.xaml.cs

```diff
@@ -1,11 +1,33 @@
 ﻿namespace Haulage;
 
+using Haulage.BaseClasses.Accounting;
+using System;
+using Microsoft.Maui.Controls;
+
 public partial class App : Application
 {
-	public App()
+    public static INavigationService NavigationService { get; private set; }
+
+    public App()
 	{
 		InitializeComponent();
 
 		MainPage = new AppShell();
-	}
+
+        NavigationService = new NavigationService();
+    }
+}
+
+public interface INavigationService
+{
+    Task NavigateToCustomerPage(Customer customer, Account account);
 }
+
+public class NavigationService : INavigationService
```

Comment:
Is the NavigationService actually used? It looks like Routing.RegisterRoute is being used instead of the NavigateToCustomerPage method being defined inside navigationService

Reply:
Totally missed this from a test on updating navigation, this has now been removed.